aries-site/src/pages/components/index.js

```diff
@@ -1,33 +1,59 @@
 import React from 'react';
-import { Layout, NavPage } from '../../layouts';
-import { DescriptiveHeader, Meta, ComingSoon } from '../../components';
-import { getPageDetails } from '../../utils';
+import Link from 'next/link';
+import { Box, Heading, Paragraph } from 'grommet';
+
+import { CardGrid, ContentCard, Meta } from '../../components';
+import { structure } from '../../data';
+import { Layout, PageIntro } from '../../layouts';
+import { getPageDetails, getParentPage, nameToPath } from '../../utils';
 
 const title = 'Components';
-const page = getPageDetails(title);
+const pageDetails = getPageDetails(title);
 
-const Components = () => {
-  const descriptiveHeader = (
-    <DescriptiveHeader
-      background={page.color}
-      subText={page.description}
-      icon={page.icon}
-      title={title}
-    />
-  );
+const cards = structure
+  .map(obj => {
+    const page = obj;
+    const parent = getParentPage(page.name);
+    page.parent = parent;
+    return page;
+  })
+  .filter(page => page.parent !== undefined)
+  .filter(page => page.parent.name === title);
 
+const Components = () => {
   return (
-    <Layout descriptiveHeader={descriptiveHeader} title={title} isNavPage>
+    <Layout title={title} isLanding>
       <Meta
         title={title}
-        description={page.seoDescription}
-        canonicalUrl="https://design-system.hpe.design/components"
+        description={pageDetails.seoDescription}
+        canonicalUrl="https://design-system.hpe.design/templates"
       />
-      {page.pages.length ? (
-        <NavPage items={page.pages} topic={page.name.toLowerCase()} />
-      ) : (
-        <ComingSoon />
-      )}
+      <Box gap="large">
+        <PageIntro
+          image={{
+            src: '/card-components.png',
+            alt: 'Card deck of HPE Design System component cards',
+            fit: 'cover',
+          }}
+        >
+          <Box justify="center" fill>
+            <Heading margin="none">{title}</Heading>
+            <Paragraph fill>{pageDetails.description}</Paragraph>
+          </Box>
+        </PageIntro>
+        <CardGrid>
+          {cards.map(topic => (
+            // Need to pass href because of: https://github.com/zeit/next.js/#forcing-the-link-to-expose-href-to-its-child
+            <Link key={topic.name} href={nameToPath(topic.name)} passHref>
```

Comment:
It seems like the designs aren't having any categorization of components anymore and that this page becomes a kitchen sink of everything. I assume for the current stage, we will just leave as is since we don't have individual pages for each component, but that in the future we'll update?

Reply:
Would be good to talk through to clarify your point here, but updating `structure.js` and creating pages for each component would be next steps in my mind.

Reply:
Cool, I think we are on the same page.